gtfu/src/main/java/gtfu/GraphicReport.java

```diff
@@ -489,11 +497,16 @@ private void drawMap(Graphics2D g, TripReportData td, int length) {
 
             Integer count = latLonMap.get(latLon).count;
             float scaledDotSize = DOT_SIZE * (1 + (count - 1) / DOT_SIZE_MULTIPLIER);
+            double alpha = Math.pow(OPACITY_MULTIPLIER, (double) (count - 1) );
+            float alphaFinal = (float) ((alpha >= ALPHA_MIN) ? alpha : ALPHA_MIN);
```

Comment:
nitpick: `alpha = Math.max(ALPHA_MIN, alpha)` is more concise.

Reply:
Wow, yeah. Thanks.